packages/backend-modules/mail/MailchimpInterface.js

```diff
@@ -90,6 +98,50 @@ const MailchimpInterface = ({ logger }) => {
         throw new NewsletterMemberMailError({ error, email })
       }
     },
+    async updateMemberInAudience(email, data, audienceId) {
```

Comment:
maybe here we could reuse `async updateMember(email, data)` but with a default audienceId as well, i.e. `async updateMember(email, data, audienceId = MAILCHIMP_MAIN_LIST_ID)` (if the rest of the logic is exactly the same)